### PR TITLE
Activate a bunch of group tests.

### PR DESCRIPTION
**Pull request by alfbr**
All group tests are checked for correctness. These are all that are correct in current master.

### CODE REVIEW

**Comment by alfbr** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
jenkins build this opm-tests=306 please

**Comment by bska** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
I think there may be a mistake here somewhere. The 'update' request doesn't generate any new files (meaning #307 is empty).

**Comment by alfbr** [comment]
Bah, you are absolutely right. My deepest apology for sending you on a wild goose chase. Should be fixed now. Thanks for your patience.

**Comment by alfbr** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
> jenkins build this update_data please

PR OPM/opm-tests#308 has data.  This looks promising.  I'll rerun this with that updated data and merge if the build check comes back green.

**Comment by bska** [comment]
jenkins build this opm-tests=308 please

File: compareECLFiles.cmake [hunk 1]
```diff
@@ -566,6 +566,34 @@ add_test_compareECLFiles(CASENAME 9_3a_grpctl_msw_model2
                          REL_TOL ${rel_tol}
                          DIR model2)
 
+add_test_compareECLFiles(CASENAME 9_3b_grpctl_stw_model2
+                         FILENAME 3B_GINJ_GAS_EXPORT_STW
```
Comment (bska) [inline compareECLFiles.cmake:570]: I believe the `FILENAME` here should be
```
9_3B_GINJ_GAS_EXPORT_STW
```
with a leading `9_` prefix.

File: compareECLFiles.cmake [hunk 2]
```diff
@@ -566,6 +566,34 @@ add_test_compareECLFiles(CASENAME 9_3a_grpctl_msw_model2
...
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR model2)
+
+add_test_compareECLFiles(CASENAME 9_3b_grpctl_msw_model2
+                         FILENAME 3B_GINJ_GAS_EXPORT_MSW
```
Comment (bska) [inline compareECLFiles.cmake:577]: I believe the `FILENAME` here should be
```
9_3B_GINJ_GAS_EXPORT_MSW
```
with a leading `9_` prefix.

File: compareECLFiles.cmake [hunk 3]
```diff
@@ -566,6 +566,34 @@ add_test_compareECLFiles(CASENAME 9_3a_grpctl_msw_model2
...
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR model2)
+
+add_test_compareECLFiles(CASENAME 9_3c_grpctl_stw_model2
+                         FILENAME 3C_GINJ_GAS_GCONSUMP_STW
```
Comment (bska) [inline compareECLFiles.cmake:584]: I believe the `FILENAME` here should be
```
9_3C_GINJ_GAS_GCONSUMP_STW
```
with a leading `9_` prefix.

File: compareECLFiles.cmake [hunk 4]
```diff
@@ -566,6 +566,34 @@ add_test_compareECLFiles(CASENAME 9_3a_grpctl_msw_model2
...
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR model2)
+
+add_test_compareECLFiles(CASENAME 9_3c_grpctl_msw_model2
+                         FILENAME 3C_GINJ_GAS_GCONSUMP_MSW
```
Comment (bska) [inline compareECLFiles.cmake:591]: I believe the `FILENAME` here should be
```
9_3C_GINJ_GAS_GCONSUMP_MSW
```
with a leading `9_` prefix.

File: compareECLFiles.cmake [hunk 5]
```diff
@@ -580,20 +608,48 @@ add_test_compareECLFiles(CASENAME 9_4a_grpctl_msw_model2
...
+
 add_test_compareECLFiles(CASENAME 9_4c_grpctl_msw_model2
                          FILENAME 9_4C_WINJ_GINJ_VREP-W_REIN-G_MSW
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
                          DIR model2)
 
+add_test_compareECLFiles(CASENAME 9_4d_grpctl_stw_model2
+                         FILENAME 9__4D_WINJ_GINJ_GAS_EXPORT_STW
```
Comment (bska) [inline compareECLFiles.cmake:640]: I believe the `FILENAME` here should be
```
9_4D_WINJ_GINJ_GAS_EXPORT_STW
```
With a single underscore (`_`) between the leading `9` and `4D`.

**Comment by alfbr** [comment]
Indeed, a couple of typos snuck in, thanks for tracking them down for me. Hopefully fixed now. Will give it a spin.

**Comment by alfbr** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
jenkins build this opm-tests=308 please

**Comment by bska** [comment]
Apparently my last attempt at a final build check picked up a stale reference to OPM/opm-tests#308.  I'll temporarily close that PR and reopen it in a bit to see if I can coax Jenkins into accepting the new state.

**Comment by bska** [comment]
jenkins build this opm-tests=308 please

**Comment by bska** [comment]
Closing and reopening OPM/opm-tests#308 was sufficient to purge the stale reference and enable us to verify that the new reference data actually allow the run to go through.  I'll merge this now.  Thanks a lot for the increased test coverage.

**Pull request merged**